Analysis/Tasks/PWGHF/HFTreeCreatorD0ToKPi.cxx

```diff
@@ -0,0 +1,263 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file HFTreeCreator2Prong.cxx
+/// \brief Writer of the 2 prong candidates in the form of flat tables to be stored in TTrees.
+///        Intended for debug or for the local optimization of analysis on small samples.
+///        In this file are defined and filled the output tables
+///
+/// \author Nicolo' Jacazio <nicolo.jacazio@cern.ch>, CERN
+
+#include "Framework/runDataProcessing.h"
+#include "Framework/AnalysisTask.h"
+#include "DetectorsVertexing/DCAFitterN.h"
+#include "AnalysisDataModel/HFSecondaryVertex.h"
+#include "AnalysisDataModel/HFCandidateSelectionTables.h"
+#include "AnalysisCore/trackUtilities.h"
+#include "ReconstructionDataFormats/DCA.h"
+
+using namespace o2;
+using namespace o2::framework;
+using namespace o2::aod::hf_cand_prong2;
+
+namespace o2::aod
+{
+namespace full
+{
+DECLARE_SOA_COLUMN(RSecondaryVertex, rSecondaryVertex, float);
```

Comment:
Why do you need to declare all these columns again? They already exist in the respective `hf_cand...` namespaces.